### PR TITLE
Offset calculation is multiplicative in GenICam + Default for increment in Integer Nodes

### PR DESCRIPTION
**Pull request by frankjannis**
- [x] Check `test_all.sh` is passed.
- [x] Add `fix #{ISSUE_NUMBER}` if the corresponding issue exists.
- [x] Fill out `## Changelog` section. If the change is for only internal use, please write `None` to the section.

## Changelog
Bugfix: The **product** of _Offset_ and _Index_ need to be added to the address for address offset calculations.
Bugfix: Default increment for Integer Nodes is 1.

### CODE REVIEW

**Comment by frankjannis** [comment]
I just noted that #182 exists. As correctly stated there, this PR does not fix all the problems but at least fixes one of them...

**Comment by frankjannis** [comment]
Hi! Something wrong with this PR or just currently too busy @Y-Nak :)?

**Comment by bschwind** [comment]
Hey @frankjannis, thanks for the PR, and sorry I missed this! The fix looks good, but it would be great to add a test that fails on `main` and gets fixed with this PR to ensure the behavior going forward.

Also, there seem to be two independent fixes here? It might be good to split them out to separate PRs if you don't mind, unless they're truly both required for the fix to make sense.

We could also help to get the other fix in, related to using the length of the register when the offset is not specified.

**Comment by frankjannis** [comment]
Hi @bschwind, thanks for responding! The 2 commits are indeed independent fixes and could be separated.
I currently don't have the time, to completely fix the behavior for the register lengths. Also, this repo does not seem that active anymore...

**Comment by bschwind** [comment]
> I currently don't have the time, to completely fix the behavior for the register lengths.

That's okay, I might take a crack at it and see what I come up with.

> Also, this repo does not seem that active anymore...

It's true, but this crate is being used in several cameras running 24/7 capture and is quite stable, so there's not a lot of activity outside of bug fixes like this one.

File: genapi/src/parser/integer.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ impl Parse for IntegerNode {
         let inc = node
             .parse_if(INC, node_builder, value_builder, cache_builder)
             .or_else(|| node.parse_if(P_INC, node_builder, value_builder, cache_builder))
-            .unwrap_or(ImmOrPNode::Imm(10));
+            .unwrap_or(ImmOrPNode::Imm(1));
```
Comment (bschwind) [inline genapi/src/parser/integer.rs:48]: I _believe_ this is correct, but is there somewhere in the spec which states this?

Comment (frankjannis) [reply genapi/src/parser/integer.rs:48]: > I _believe_ this is correct, but is there somewhere in the spec which states this?

https://www.emva.org/wp-content/uploads/GenICam_Standard_v2_1_1.pdf
Page 51. There are the defaults stated.

**Review by bschwind — APPROVED** [review]
LGTM! Sorry for taking so long on a simple change, I kind of let this one get lost in my inbox.

**Pull request merged**